constants/index.ts

```diff
@@ -78,26 +82,26 @@ export const skills = [
 
 export const testimonials = [
   {
-    name: "First",
+    name: "Tye",
     quote:
-      "I have had the pleasure of working with Jay as a Software Developer and can confidently say that he is a valuable asset to any team. He is highly skilled in ReactJS NextJS, Javascript, Material UI, Tailwind CSS, Redux, and GitHub, and consistently delivers high-quality code",
-    title: "Creator of Nothing",
+      "One of Jay's standout qualities is his ability to convey ideas and requirements in a clear and concise manner. This greatly facilitated our work together, ensuring that we were always on the same page regarding project objectives and tasks. Jay's excellent communication skills were a cornerstone for efficient collaboration, making it easy to address any challenges or changes that arose during the development process. Overall, I highly recommend Jay as a developer and a valued team member.",
```

Comment:
Why isn't this made through sanity?

Reply:
@Nassim-Saboundji it will be but I realized I made a type change before that was breaking my site so I wanted to push these changes live now and then open another branch to add this to sanity